### ISSUE TITLE
Parameter Settings -debug-duration & -debug-interval Do Not pass to -nomad

### ISSUE THREAD

**Issue by MikaelHashiCorp** [hashicorp/hcdiag#268]
**What is the bug?**  
When one sets the `-debug-duration` & `-debug-interval` parameters, they are not passed to `nomad operator debug`.

**What version are you using?**  
hcdiag v0.4.0 

**Which HashiCorp products does this affect?**  
hcdiag & nomad

**When does the error occur?**  
1. Have Nomad and Consul agents running.
2. Download latest versions of consul, hcdiag, and nomad.
3. Set env path:  `export PATH=$PWD:$PATH`
4. Run the following command: ```./hcdiag -consul -nomad -debug-duration 10s -debug-interval 05s```
5. Unzip the hcdiag bundle.
6. Look in file `results.json` and see:  
```"nomad operator debug -log-level=TRACE -node-id=all -max-nodes=10 -output=/home/ubuntu/shared/diag/hcdiag-2022-11-08T203755Z388856484 -duration=2m0s -interval=30s"```
8. Notice that `-duration` _doesn't_ equal 10s and `-interval` _doesn't_ equal 5s. 
9. Look in file `results.json` and see:  
`"consul debug -output=/home/ubuntu/shared/diag/hcdiag-2022-11-08T203755Z388856484/ConsulDebug -duration=10s -interval=5s"`
10. Notice that `-duration` does equal 10s and `-interval` does equal 5s.
11. Conclude that the parameters were passed correctly to `consul debug`, but not `nomad operator debug`.

**What is the expected behavior?**  
The correct `-duration` and `-interval` parameters get passed to `nomad operator debug`.

**Is there a workaround?**  
No.

**How does this impact you?**  
In addition to running `hcdiag`, I will need to run `nomad operator debug` at customer sites, doubling the work I am asking them to do.

**Are there any deadlines?**  
No.

**Is there any additional context?**  
No.

**Comment by mallika-hc** [comment]
In addition to be able to specify `debug duration` and `debug interval`, I would like to request that we be able to specify the `metrics interval` and also be able to specify if we want to set `compress=false` .  More details [here](https://developer.hashicorp.com/vault/docs/commands/debug#:~:text=Start%20debug%20with%20different,%24)
Let me know if you want me to create a Jira for this @aliciaclark1066 Thanks!

**Comment by MikaelHashiCorp** [comment]
Please provide a status to this issue.

**Comment by aliciaclark1066** [comment]
@MikaelHashiCorp Yes, we've addressed this issue and will release the fix in v0.5.0.

**Comment by MikaelHashiCorp** [comment]
**User acceptance testing (UAT) this issue against `hcdiag v0.5.0-rc2`**

**Test summary**

Following the directions in [Custom Configuration](https://github.com/hashicorp/hcdiag/blob/main/docs/custom-config.md), the fix worked.

**Test results**

Configuration file `hcdiag.hcl`

```
product "consul" {
  consul-debug {
    archive = "true"  // The argument \"archive\" is required.
    duration = "10s"
    interval = "05s"
  }
}

product "nomad" {
  nomad-debug {
    duration = "10s"
    interval = "05s"
  }
}
```

hcdiag command:

```
hcdiag run -consul -nomad -config hcdiag.hcl
```

Select `results.json` output

```
...
  "consul": {
    "ConsulDebug": {
        "duration": "10s", 
        "interval": "05s", 
...
  "nomad": {
    "NomadDebug": {
      "error": "", 
      "status": "success", 
      "params": {
        "duration": "10s", 
        "event_topic": null, 
        "interval": "05s", 
...
```
